website/docs/r/branch_default.html.markdown

```diff
@@ -0,0 +1,37 @@
+---
+layout: "github"
+page_title: "GitHub: github_branch_default"
+description: |-
+  Provides a GitHub branch default for a given repository.
+---
+
+# github_branch_default
+
+Provides a GitHub branch default resource.
+
+This resource allows you to set the default branch for a given repository. 
+
+## Example Usage
```

Comment:
Given the potential pitfalls of requiring the repo and branch to exist before this can use it would be nice to have a more complete example highlighting what this workflow looks like for anything not matching the github created based on (now) org settings or default github and `auto_init`.

Reply:
Good point, have added a more complete example.